iotedgedev/modules.py

```diff
@@ -138,32 +138,25 @@ def build_push(self, template_file, default_platform, no_build=False, no_push=Fa
         # sample: 'localhost:5000/filtermodule:0.0.1-amd64'
         tags_to_build = set()
 
+        deployment_manifest = DeploymentManifest(self.envvars, self.output, self.utility, template_file, True)
+
+        # get image tags for ${MODULES.modulename.xxx} placeholder
         if os.path.isdir(self.envvars.MODULES_PATH):
             for module_name in os.listdir(self.envvars.MODULES_PATH):
```

Comment:
so it is still use module folder name as module name?

Reply:
We cannot use regex to grab module name from the ${MODULES.xxx} syntax, so we still need to assume the folder name is module name and user need to write the place holder in this format ${MODULES.FOLDER_NAME.PLATFORM}.
The BYPASS MODULES is filtering the key of modules in deployment template as we synced offline.